src/AppContests.tsx

```diff
@@ -22,6 +25,13 @@ export const AppContests = () => {
     },
   ]
 
+  if (scoreboard) {
```

Comment:
I think we should also check for `length===0` as it comes as an array now

Reply:
yep, good idea